src/content/docs/en/reference/experimental-flags/csp.mdx

```diff
@@ -265,6 +265,51 @@ export default defineConfig({
 });
 ```
 
+### `fontDirectiveResources`
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `[]`<br />
```

Comment:
Didn't we run into a similar tricky situation documenting something not too long ago where adding any value overrides (and removes) the default that needs to be added back in? 😅  What was that and how did we handle that?

I think what feels off here is that if you do nothing, by default you get the same result as if you added this property with `'self'`, right? So it's kind of like the default value is `'self'` and not nothing?